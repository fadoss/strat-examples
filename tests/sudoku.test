games/sudoku

srew sudoku(
	0 ; 6 ; 2 ; 	3 ; 4 ; 0 ; 	7 ; 5 ; 0 ;
	1 ; 0 ; 0 ; 	0 ; 0 ; 5 ; 	6 ; 0 ; 0 ;
	5 ; 7 ; 0 ; 	0 ; 0 ; 0 ; 	0 ; 4 ; 0 ;
	0 ; 0 ; 0 ; 	0 ; 9 ; 4 ; 	8 ; 0 ; 0 ;
	4 ; 0 ; 0 ; 	0 ; 0 ; 0 ; 	0 ; 0 ; 6 ;
	0 ; 0 ; 5 ; 	8 ; 3 ; 0 ; 	0 ; 0 ; 0 ;
	0 ; 3 ; 0 ; 	0 ; 0 ; 0 ; 	0 ; 9 ; 1 ;
	0 ; 0 ; 6 ; 	4 ; 0 ; 0 ; 	0 ; 0 ; 7 ;
	0 ; 5 ; 9 ; 	0 ; 8 ; 3 ; 	2 ; 6 ; 0 )
	 using solve .
