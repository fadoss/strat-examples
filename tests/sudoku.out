==========================================
srewrite in STRAT : sudoku(0 ; 6 ; 2 ; 3 ; 4 ; 0 ; 7 ; 5 ; 0 ; 1 ; 0 ; 0 ; 0 ;
    0 ; 5 ; 6 ; 0 ; 0 ; 5 ; 7 ; 0 ; 0 ; 0 ; 0 ; 0 ; 4 ; 0 ; 0 ; 0 ; 0 ; 0 ; 9 ;
    4 ; 8 ; 0 ; 0 ; 4 ; 0 ; 0 ; 0 ; 0 ; 0 ; 0 ; 0 ; 6 ; 0 ; 0 ; 5 ; 8 ; 3 ; 0 ;
    0 ; 0 ; 0 ; 0 ; 3 ; 0 ; 0 ; 0 ; 0 ; 0 ; 9 ; 1 ; 0 ; 0 ; 6 ; 4 ; 0 ; 0 ; 0 ;
    0 ; 7 ; 0 ; 5 ; 9 ; 0 ; 8 ; 3 ; 2 ; 6 ; 0) using solve .

Solution 1
rewrites: 1495309
result Configuration: 
< id(1, 0) : rows |	val : (9 ; 6 ; 2 ; 3 ; 4 ; 1 ; 7 ; 5 ; 8)	> 
< id(2, 0) : rows |	val : (1 ; 4 ; 8 ; 9 ; 7 ; 5 ; 6 ; 2 ; 3)	> 
< id(3, 0) : rows |	val : (5 ; 7 ; 3 ; 2 ; 6 ; 8 ; 1 ; 4 ; 9)	> 
< id(4, 0) : rows |	val : (3 ; 2 ; 1 ; 6 ; 9 ; 4 ; 8 ; 7 ; 5)	> 
< id(5, 0) : rows |	val : (4 ; 8 ; 7 ; 5 ; 1 ; 2 ; 9 ; 3 ; 6)	> 
< id(6, 0) : rows |	val : (6 ; 9 ; 5 ; 8 ; 3 ; 7 ; 4 ; 1 ; 2)	> 
< id(7, 0) : rows |	val : (8 ; 3 ; 4 ; 7 ; 2 ; 6 ; 5 ; 9 ; 1)	> 
< id(8, 0) : rows |	val : (2 ; 1 ; 6 ; 4 ; 5 ; 9 ; 3 ; 8 ; 7)	> 
< id(9, 0) : rows |	val : (7 ; 5 ; 9 ; 1 ; 8 ; 3 ; 2 ; 6 ; 4)	>

No more solutions.
rewrites: 1495309
Bye.
